whylog/config/parsers.py

```diff
@@ -113,6 +135,7 @@ class ConcatenatedRegexParser(AbstractParserSubset):
 
     def __init__(self, parser_list):
         self._parsers = parser_list
+        self._parsers_dict = dict((parser.name, parser) for parser in self._parsers)
```

Comment:
In my code I use [] on self._parsers for some range [l,r]. If I used OrderedDict I'd have to iterate over [0, r] range. OrederedDict doesn't have a function, which returns key inserted at exact moment without interating.